src/main/java/winston/DateHandler.java

```diff
@@ -0,0 +1,31 @@
+package winston;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A class that handles parsing and formatting of dates
+ */
+public class DateHandler {
+    public DateHandler() {
+    }
+
+    /**
+     * A method that will convert a date in strong format to LocalDate format
+     * @param str String of the date
```

Comment:
Maybe an empty line should be added between the description and param section for java docs. There are a few other files where this empty line should be added.

```suggestion
     * A method that will convert a date in strong format to LocalDate format.
     *
     * @param str String of the date.
```